NCS.DSS.EmploymentProgression/GetEmploymentProgressionById/Service/EmploymentProgressionGetByIdTriggerService.cs

```diff
@@ -5,10 +5,10 @@ namespace NCS.DSS.EmploymentProgression.GetEmploymentProgressionById.Service
 {
     public class EmploymentProgressionGetByIdTriggerService : IEmploymentProgressionGetByIdTriggerService
     {
-        private readonly IDocumentDBProvider _documentDbProvider;
-        private readonly IServiceBusClient _serviceBusClient;
+        private readonly ICosmosDBProvider _documentDbProvider;
```

Comment:
Can we rename this to cosmosDBProvider please as we no longer use documentDBProvider